### PR TITLE
Fix notv placement

### PR DESCRIPTION
**Pull request by SpotComms**
- Fix notv placement
- Fix "include include" in mate-calculator.profile a1cb05b0304c0f0fb68582a34fcfcfe1d4b505f0
- Fix "nosound\nnosound" in 7z.profile
- Fix options out of alphabetical order in default.profile

### CODE REVIEW

File: etc/7z.profile [hunk 1]
```diff
@@ -13,11 +13,11 @@ net none
 no3d
 nosound
 nosound
```
Comment (reinerh) [inline etc/7z.profile]: Just noticed here "nosound" is duplicated. Can you please remove the line?

File: etc/Cyberfox.profile [hunk 2]
```diff
@@ -2,5 +2,4 @@
 # This file is overwritten after every install/update
 
 
-# Redirect
```
Comment (reinerh) [inline etc/Cyberfox.profile]: Why remove those? Were they not recently added intentionally?

Comment (SpotComms) [inline etc/Cyberfox.profile]: Alias profiles are already marked with "Firejail profile alias for x"

File: etc/7z.profile (hunk 1 above)
Comment (SpotComms) [inline etc/7z.profile]: Fixed

File: etc/Cyberfox.profile (hunk 2 above)
Comment (netblue30) [inline etc/Cyberfox.profile]: Add it back.  I need "redirect" as a marker to add new commands globally, and some redirected profiles are not aliases. For example thunderbird redirects to firefox. I'll be bringing in some more, thanks!

Comment (SpotComms) [inline etc/Cyberfox.profile]: @netblue30 Added back.

**Comment by Fred-Barclay** [comment]
Merged, thanks!

**Pull request merged**